pkg/scheduler/plugins/coscheduling/core/core.go

```diff
@@ -311,16 +328,40 @@ func (pgMgr *PodGroupManager) Unreserve(ctx context.Context, state *framework.Cy
 	gang.delAssumedPod(pod)
 
 	if !gang.isGangOnceResourceSatisfied() && gang.getGangMode() == extension.GangModeStrict {
-		// release resource of all assumed children of the gang
+		pgMgr.rejectGangGroupById(gang.Name, handle)
+	}
+}
+
+func (pgMgr *PodGroupManager) rejectGangGroupById(gangId string, handle framework.Handle) {
```

Comment:
Maybe can add pluginName string as input, print waitingPod.Reject(pluginName,"gang rejection by another ganggroup").
Or modify the interface Manager's functions Unreserve、PostFilter.

Reply:
The pgMgr.PostFilter method already has a pluginName parameter. Can use this directly.

Reply:
good idea, thanks